cpp/src/prims/key_store.cuh

```diff
@@ -147,14 +148,15 @@ class key_cuco_store_view_t {
 
   static constexpr bool binary_search = false;
 
-  using cuco_store_type = cuco::experimental::static_set<
-    key_t,
-    cuco::experimental::extent<std::size_t>,
-    cuda::thread_scope_device,
-    thrust::equal_to<key_t>,
-    cuco::experimental::linear_probing<1,  // CG size
-                                       cuco::murmurhash3_32<key_t>>,
-    rmm::mr::stream_allocator_adaptor<rmm::mr::polymorphic_allocator<std::byte>>>;
+  using cuco_store_type =
```

Comment:
Now we have both `cuco_storage_type` and `cuco_store_type` and I assume this can be confusing. Should we better rename `cuco_store_type` to something else (e.g. `cuco_set_type`?) to avoid confusion?

Reply:
Good point. Fixed.